src/main/java/com/dev/springapp/controller/UserController.java

```diff
@@ -0,0 +1,53 @@
+package com.dev.springapp.controller;
+
+import com.dev.springapp.dto.UserResponseDto;
+import com.dev.springapp.model.User;
+import com.dev.springapp.service.UserService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+    @Autowired
+    UserService userService;
```

Comment:
???

Reply:
If issue related to access modifier - fixed that